e2eTests/removeElements/removeElementsConfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "gridUrl": "http://localhost:4444/wd/hub",
+  "baseline": "./e2eTests/removeElements/baselineMockImages",
+  "latest": "./e2eTests/removeElements/latestMockImages",
+  "generatedDiffs": "./e2eTests/removeElements/generatedDiffs",
+  "report": "./e2eTests/removeElements/reports",
+  "remoteBucketName": "aye-spy-e2e-tests",
+  "remoteRegion": "eu-west-1",
+  "scenarios": [
+    {
+      "url": "https://s3-eu-west-1.amazonaws.com/aye-spy-e2e-tests/chrome/report/index.html",
```

Comment:
We could host this all locally. Add an NGINX image to your compose file.

Reply:
boooo
lame

Reply:
done